### ISSUE TITLE
Client.Timeout exceeded while awaiting headers

### ISSUE THREAD

**Issue by rafrox** [appbaseio/abc#89]
abc version 0.5.0

Hey guys, after running the command line (windows and ubuntu as well):

abc-0.5.0 import --src_type=mysql --src_uri="USER:PASS@tcp(XXXXXXX.amazonaws.com:3306)/MY_DB_NAME" "https://MY_ADMIN_API_KEY@scalr.api.appbase.io/raftest"

I get the following error and no data indexed:

[31mERRO[0m[0065] Post https://scalr.api.appbase.io/raftest/_bulk: net/http: request canceled (Client.Timeout exceeded while awaiting headers)  [31mversion[0m=2 [31mwriter[0m=elasticsearch
[31mERRO[0m[0065] Post https://scalr.api.appbase.io/raftest/_bulk: net/http: request canceled (Client.Timeout exceeded while awaiting headers)  [31mname[0m=sink [31mpath[0m="source/sink" [31mtype[0m=elasticsearch
2000 total data record(s) indexed
[31mERRO[0m[0075] timeout reached waiting for Out channels to clear
panic: send on closed channel

goroutine 25 [running]:
github.com/appbaseio/abc/private/adaptor/mysql.(*Reader).iterateTable.func1(0xc0420388a0, 0xc042038840, 0xc042395cc0, 0xc0423b0000, 0xc042038660, 0xc042038b40)
        /Users/siddharthlatest/.gvm/pkgsets/go1.9/global/src/github.com/appbaseio/abc/private/adaptor/mysql/reader.go:148 +0x2e5
created by github.com/appbaseio/abc/private/adaptor/mysql.(*Reader).iterateTable
        /Users/siddharthlatest/.gvm/pkgsets/go1.9/global/src/github.com/appbaseio/abc/private/adaptor/mysql/reader.go:99 +0x95

##### Checks

- [X ] I have mentioned my `abc` version in the issue, if applicable.
- [X ] This issue is not a duplicate.
- [X ] This issue has a description.

**Comment by siddharthlatest** [comment]
@rafrox Thanks! We will try to replicate this with MySQL hosted on AWS and get back.

**Comment by rafrox** [comment]
Let me know when you have news. I am really excited to use ABC since this is the last piece I need to connect my MySql to appbase and fully automate my site :)

**Comment by siddharthlatest** [comment]
@rafrox Were you able to get this to work? This most likely seems like an issue with network connectivity being intermittent.

**Comment by rafrox** [comment]
> @rafrox Were you able to get this to work? This most likely seems like an issue with network connectivity being intermittent.

Yes and no. So error seems to be random. Some times it worked and sometimes not. I have tried to change from my main internet (cable) to a 4g modem I have.. same thing (sometimes works sometimes not).

Also another thing is that for some reason importing data using ABC is duplicating data into appbase. I have first created a small amount of records using regular process via "import CSV" in the appbase platform just to make sure everything was setup nice. After that started using ABC but entries got all duplicated every time I import (uniqueID is not being respected I dont know why).
